ci_scripts/ci-build.sh

```diff
@@ -25,20 +19,13 @@ if [ -z "$TRAVIS_OS_NAME" ]; then
             exit 1
     esac
 fi
-SYS_DC_CORE=${SYS_DC_CORE:-true}
 
+SYS_DC_CORE=${SYS_DC_CORE:-false}
 
-if [ $TRAVIS_OS_NAME = linux ]; then
-    CONTAINER_ID=${CONTAINER_ID:-$(docker ps --latest --format='{{.ID}}')}
-    EXEC="docker exec $CONTAINER_ID";
```

Comment:
On one side i like removing docker because it makes the ci run faster, on the other hand i also like the docker layer because it gives a certain reproducability which is otherwise hard on travis and running it locally. Don't have strong opinions on this, just want to point that out.

Reply:
I hear you. I also like docker but right now it's not really needed since we basically care about `cargo` and a c compiler.